src/multiplayer-servers/getting-started/sidebar.json

```diff
@@ -4,11 +4,15 @@
   "order": 1,
   "items": [
     {
-      "text":"Introduction",
+      "text": "Introduction",
       "link": "/getting-started/introduction"
     },
     {
-      "text":"Authentication",
+      "text": "Setup your Environment",
```

Comment:
I feel like this should go after `Pushing Container Images` or at least `Authentication` as these are prerequisite steps